### ISSUE TITLE
Variables are invalid JSON

### ISSUE THREAD

**Issue by vincenzo** [prismagraphql/graphql-playground#308]
#### This issue pertains to:

- [X] GraphQL Playground Middleware

#### What OS and OS version are you experiencing the issue(s) on?

macOS High Sierra 10.13.1

#### What version of graphql-playground(-electron/-middleware) are you experience the issue(s) on?

1.3.4

#### What is the expected behavior?

Valid response

#### What is the actual behavior?

Error: "Variables are invalid JSON"

#### What steps may we take to reproduce the behavior?

1. Create your GraphQL server with Express and Apollo (get off the ground with [graphql-yoga](https://github.com/graphcool/graphql-yoga), if you like)
2. Include Playground (already done for you if you chose to start with graphql-yoga)
3. Load PG and Hit the "copy curl" for any of your queries
4. Run the curl line

My example line: 

```bash
curl 'http://localhost:3005/graphql' -H 'Origin: http://localhost:3005' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: */*' -H 'Connection: keep-alive' -H 'DNT: 1' --data-binary '{"query":"{\n\tfilms {\n    id\n  }\n}","variables":"","operationName":null}' --compressed
```

**Note**: 

Replacing `"variables": ""` with `"variables":null` fixes the issue. Removing the variables part altogether also fixes it.

**Note again**: 

The issue does not currently occur with graph.cool, and by this I mean that the `"variables": ""` is there, but then making the curl request to the graph.cool service, the server doesn't respond with the same error, but it just returns a valid response.

**Comment by arquio** [comment]
Same problem

**Comment by lukehedger** [comment]
This has been fixed in 0eb2492545d8c58de80eb8e435840e3fadc144b2 - just tested with v1.3.22 and 'Copy CURL' is producing a valid command.

**Comment by timsuchanek** [comment]
Fixed in the latest version. Please let us know if you still have that issue, then we reopen it!
